sos/report/__init__.py

```diff
@@ -652,13 +652,18 @@ def _set_tunables(self):
     def _check_for_unknown_plugins(self):
         import itertools
         for plugin in itertools.chain(self.opts.only_plugins,
-                                      self.opts.skip_plugins,
                                       self.opts.enable_plugins):
             plugin_name = plugin.split(".")[0]
             if plugin_name not in self.plugin_names:
                 self.soslog.fatal('a non-existing plugin (%s) was specified '
                                   'in the command line' % (plugin_name))
                 self._exit(1)
+        for plugin in self.opts.skip_plugins:
+            if plugin not in self.plugin_names:
+                self.soslog.warning(
+                    "Requested to skip non-existing plugin '%s'" % plugin
+                )
+        self.ui_log.info('')
```

Comment:
Minor comment 2: this will add the empty line unconditionally, to:

    sosreport (version 4.1)


    This command will collect diagnostic and configuration information from
    this Red Hat Enterprise Linux system and installed applications.

    An archive containing the collected information will be generated in
    ..

I would prefer adding the line only when a warning has been printed.

Reply:
Yeah, I was trying to avoid tracking reports just to print a blank line, but the way the policy banner msg is built and printed makes this difficult to do neatly. I'm leaning towards just dropping the empty line and letting the banner just immediately follow any printed warning (as adding a newline instead makes multiple 'bad' plugins super spaced out)